_pages/dev/ssr/server-side-rendering-contextual-logging.md

```diff
@@ -313,39 +317,50 @@ The following is an example of such a log:
 }
 ```
 
-Depending on the error ocurred during the parser, following error messages may be generated:
-- `Traceparent header has invalid length: ${traceparent.length}. Expected 55 characters.` - the `traceparent` header has invalid length
-- `Traceparent header has invalid format.` - the `traceparent` header has invalid format. To learn more, see [W3C Trace Context - Traceparent Header](https://www.w3.org/TR/trace-context/#traceparent-header)
-
```

Comment:
I removed this part because the explanation that was provided about each error message was the same as the error message itself. And as for the link, it leads to a section of a page that shows what the fields are for the `transparent` header, but we already provide a link to that page at the start of the trace context section, so I didn't see the value in putting any of this information here....